Lib/Stack.hpp

```diff
@@ -371,12 +371,12 @@ class Stack
 
   template<class Less = std::less<C>>
   void sort(Less less = std::less<C>{})
-  { std::sort(begin(), end(), less); }
+  { std::sort(begin(), end(), std::move(less)); }
 
   /** like Stack::sort but moves the content out of `this` and returns the resulting Stack instead of changing the contents of this */
-  template<class Equal = std::equal_to<C>>
-  Stack sorted(Equal eq = std::equal_to<C>{})
-  { sort(); return std::move(*this); }
+  template<class Less = std::less<C>>
+  Stack sorted(Less less = std::less<C>{})
+  { sort(std::move(less)); return std::move(*this); }
```

Comment:
Weird. Did this ever work?

Reply:
No because I always put an explicit argument there :)